class-components/src/components/Card/__tests__/Card.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { describe, it, expect, vi } from 'vitest';
+import Card from '../Card';
+import { ISingleResult } from '../../../types/AppTypes';
+import '@testing-library/jest-dom';
+
+const renderCard = (result: ISingleResult, onClick = vi.fn()) => {
+  return render(<Card result={result} onClick={onClick} />);
+};
+
+describe('Card component', () => {
+  it('renders the real card data', () => {
+    const result: ISingleResult = {

```

Comment:
you can use vitest `beforeall` or `beforeeach` functions to define this test object instance instead of repeating code